crypto/ec/ec_ameth.c

```diff
@@ -43,7 +43,17 @@ static int eckey_param2type(int *pptype, void **ppval, EC_KEY *ec_key)
         pstr = ASN1_STRING_new();
         if (pstr == NULL)
             return 0;
-        pstr->length = i2d_ECParameters(ec_key, &pstr->data);
+
+        /*
+         * The cast in the following line is intentional as the
+         * `i2d_ECParameters` signature can't be constified (see discussion at
+         * https://github.com/openssl/openssl/pull/9347 where related and
+         * required consitfication backports were rejected).
```

Comment:
typo, s/consitfication/constfication/  ? I am not sure of that is an english word...

Reply:
Thanks! Fixing the typo now.

It might not be on official British or American English dictionaries, but I believe it is a widely-used neologism in developer lexicon (a quick google search shows that it is used on Linux kernel development and other big projects).